recipe/meta.yaml

```diff
@@ -26,9 +26,18 @@ test:
     - sphinx_bootstrap_theme
 
 about:
-  home: http://ryan-roemer.github.com/sphinx-bootstrap-theme/README.html
-  license: MIT License
+  home: http://ryan-roemer.github.com/sphinx-bootstrap-theme/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Normally this would require a build number bump, but the rename has the same effect.